pkg/arm/data/master-startup.sh

```diff
@@ -659,6 +659,16 @@ sed -i -re "s#( *server: ).*#\1https://$(hostname)#" /etc/origin/node/node.kubec
 # daemonset will not fail and set the master node labels correctly.
 cp /etc/origin/node/node.kubeconfig /etc/origin/node/bootstrap.kubeconfig
 
+{{- if $.Extra.IsRecovery }}
+tempBackDir=$(mktemp -d -t etcd-backup)
+docker run -v /etc/origin/cloudprovider/:/_data/_out -v $tempBackDir:/out:z {{ .Config.Images.GetBackup }} --blobname=$.Extra.BackupBlobName --destination=/out/backup.db
```

Comment:
@aaronlevy fyi, this is a proposed solution for how we are going to restore from backups. Is this something handled during Ignition in the 4.x installer today? If not, how are you thinking of doing restores?